lib/i18n/tasks/ar/model.rb

```diff
@@ -4,6 +4,7 @@ module Ar
       module Model
         class << self
           def names
+            Rails.application.eager_load! if Rails.env == 'development' rescue true
```

Comment:
Line is too long. [83/80]<br>Avoid using `rescue` in its modifier form.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.